siren/websocket.go

```diff
@@ -17,25 +17,9 @@ var upgrader = websocket.Upgrader{
 	WriteBufferSize: 1024,
 }
 
-// from us to the client
-type WSMsg struct {
-	Type string      `json:"type"`
-	ID   string      `json:"id,omitempty"` // in response to something
-	Msg  interface{} `json:"msg"`
-}
-
-// from the client to us. Cmd is always filled.
-type WSCmd struct {
-	Cmd     string  `json:"cmd"`
-	ID      string  `json:"id"` // will be used as WSMsg.ID
-	What    string  `json:"what"`
-	Artist  string  `json:"artist"`
-	Album   string  `json:"album"`
-	Track   string  `json:"track"`
-	File    string  `json:"file"`
-	Seconds float64 `json:"seconds"`
-	Song    string  `json:"song"`
-	Volume  float64 `json:"volume"`
+type WSMsg interface {
+	isMsg()
```

Comment:
maybe drop that? Seems not to really add value here.

Reply:
just the `isMsg`? because we only use the `Type() string`?

Reply:
yes. The isMsg is just Go type magic.